src/store/reducers/movieReducer.js

```diff
@@ -0,0 +1,61 @@
+import * as MovieActionTypes from '../actions/movie/movieActionTypes.js';
+
+const initialState = {
+  movies: [],
+  movieForDeletion: null,
+  moviePreview: null,
+  movieForEdit: null,
+  totalMoviesCount: 0,
+  genres: ['All'],
+  genreForFilter: 'All',
+  orderForSorting: 'title' 
+};
+
+const movieReducer = (state = initialState, action) => {
```

Comment:
the same for reducer, you should have several and combine them

Reply:
move to common